.env.sample

```diff
@@ -0,0 +1,2 @@
+REACT_APP_FARM_ADDRESS=0xdb7d0a3bfcb7b34e6be510b26c55c8b60bd66d4a
```

Comment:
No problem with actually leaving the farm address exposed—that's actually good. We just don't want everybody hooking up to the subgraph endpoint haha